packages/api-gateway/test/auth.e2e-spec.ts

```diff
@@ -0,0 +1,158 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import {
+  ClassSerializerInterceptor,
+  INestApplication,
+  ValidationPipe,
+} from '@nestjs/common';
+import { AppModule } from './../src/app.module';
+import { Reflector } from '@nestjs/core';
+import * as request from 'supertest';
+import { ResetProtoFile } from 'juno-proto';
+import * as GRPC from '@grpc/grpc-js';
+import * as ProtoLoader from '@grpc/proto-loader';
+
+let app: INestApplication;
+const ADMIN_EMAIL = 'test-superadmin@test.com';
+const ADMIN_PASSWORD = 'test-password';
+
+beforeAll(async () => {
+  const proto = ProtoLoader.loadSync([ResetProtoFile]) as any;
+
+  const protoGRPC = GRPC.loadPackageDefinition(proto) as any;
+  const resetClient = new protoGRPC.juno.reset_db.DatabaseReset(
+    process.env.DB_SERVICE_ADDR,
+    GRPC.credentials.createInsecure(),
+  );
+  await new Promise((resolve) => {
+    resetClient.resetDb({}, () => {
+      resolve(0);
+    });
+  });
+});
+
+afterAll((done) => {
+  app.close();
+  done();
+});
+
+beforeEach(async () => {
+  const moduleFixture: TestingModule = await Test.createTestingModule({
+    imports: [AppModule],
+  }).compile();
+
+  app = moduleFixture.createNestApplication();
+  app.useGlobalPipes(
+    new ValidationPipe({
+      transform: true,
+    }),
+  );
+  app.useGlobalInterceptors(new ClassSerializerInterceptor(app.get(Reflector)));
+
+  await app.init();
+});
+
+describe('Invalid key push to get jwt route', () => {
```

Comment:
You can safely ignore this comment if we use this naming pattern elsewhere, but I would avoid categorizing describe tests as valid or invalid tests, and instead categorize based on the component tested (e.g. auth jwt routes)